src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/util/ManagedIndexUtils.kt

```diff
@@ -175,3 +178,43 @@ fun getSweptManagedIndexSearchRequest(): SearchRequest {
                     )
                     .query(boolQueryBuilder))
 }
+
+enum class ActionType(val type: String) {
```

Comment:
Any reason this is not a file on its own? If we want it to be an inner class I think it fit better in `ActionConfig` to hold all the related Action data into it.

Reply:
Moved into ActionConfig